connector_search_engine/tools/adapter.py

```diff
@@ -2,18 +2,12 @@
 # Sébastien BEAU <sebastien.beau@akretion.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo.addons.component.core import AbstractComponent
 
-
-class SeAdapter(AbstractComponent):
-    _name = "se.backend.adapter"
-    _inherit = ["base.se.connector", "base.backend.adapter"]
-    _usage = "se.backend.adapter"
-
-    @classmethod
-    def match(cls, session, model):
-        # We are a generic exporter; how cool is that?
-        return True  # pragma: no cover
+class SearchEngineAdapter:
+    def __init__(self, backend_record, index_record):
+        super().__init__()
+        self.backend_record = backend_record
+        self.index_record = index_record
 
     def index(self, datas):
         raise NotImplementedError()
```

Comment:
```suggestion
    @abstractmethod
    def index(self, datas):
        ...
```

Be carefull the three dots are to be preserved...